Add details about new education features (#220)
GDevelopApp/GDevelop-documentation: docs/gdevelop5/education/index.md

- GDevelop gaming page.

Student accounts also have access to the version history feature for Cloud projects: they can go back to any previous save they made.

## Manage seats

From the "Manage Seats" page, it is possible to administrate your plan.

### Change account password

Students don't have the possibility to change the password of their accounts.
But teacher accounts can do it. The password is then safely stored in our database so that you can easily retrieve it.

![Zoom on the "Change password" feature](./change-password.png)

### Add teacher accounts

You can have multiple accounts having the teacher role in your plan.
To do so, click the "Add teacher" button and enter the email address of the other teacher GDevelop account.

> Note that the other teacher must already have signed up with GDevelop before you can add them to your account as teachers.

Reviewer: ```suggestion
Note that teachers must already have signed up with GDevelop before you can add them to your account. We encourage every teacher to sign up with their work/education email.
```

![Zoom on the "Add teacher" feature](./add-teacher.png)

> Teacher accounts added this way will have the same rights as the main account except for the subscription management, they cannot unsubscribe from the education plan.

Reviewer: Simplify:
```suggestion
Teacher accounts have the same rights as the main account but cannot unsubscribe from the education plan.
```

### Add, archive and restore student accounts

> Once your plan is all set up, you will have to create the student accounts yourself.

Reviewer: Too negative. 

"Once you've purchased GDevelop for Education, head to the classroom tab and click on "Manage Seats" to start creating accounts for your students."

> Using this input, create a batch of student accounts.

Reviewer: input is too technical. Use "button" or "field".

![Zoom on the "Add students batch" feature](./add-multiple-students.png)

It is then possible to manage the student accounts by archiving and restoring accounts.

> You can also copy-paste the accounts credentials in CSV format.

Reviewer: You can also export account credentials in a CSV file (that can be opened using any spreadsheet app).

![Zoom on the "Manage students" features](./manage-students.png)

Here is the recommended way of managing student accounts:

- When your account is set up, add the needed teacher accounts.

> - Then create student accounts in a batch (it should correspond to the number of remaining seats).
> - Enjoy and teach class!

Reviewer: ```suggestion
- Then create student accounts - very often, you can create all accounts in a batch, with the number of remaining seats in your account. If you create two many accounts, you can always disable them later.
- You're ready to teach! Send the credentials for each account to your students and make sure to read our curriculum and educational resources.
```
